src/agent.py

```diff
@@ -106,7 +106,7 @@ def get_move(self, neighbors: list[Agent],
         if self._body_temp[self._body_radius-1][self._body_radius-1] < self._low_move_threshold:
             target_pos = target_pos + self.position
         elif self._body_temp[self._body_radius-1][self._body_radius-1] > self._high_move_threshold:
-            target_pos = -1*target_pos + self.position
```

Comment:
@Siddharth-Barve What is this change for? Was this a mistake? This doesn't seem to pertain to the thermal model.